fauxfactory/__init__.py

```diff
@@ -520,25 +521,49 @@ def gen_negative_integer():
     return gen_integer(max_value=max_value)
 
 
-def gen_ipaddr(ip3=False, ipv6=False):
+def gen_ipaddr(ip3=False, ipv6=False, prefix=None):
     """Generates a random IP address.
 
+    You can also specify an IP address prefix if you are interested eg. in
```

Comment:
I'd remove the `eg.` here and add a `,` right before the `etc` as such: `generation, etc.`.